crates/ironrdp-pdu/src/rdp/vc/dvc/gfx/graphics_messages/avc_messages.rs

```diff
@@ -6,7 +6,7 @@ use bitflags::bitflags;
 use byteorder::{LittleEndian, ReadBytesExt, WriteBytesExt};
 
 use super::GraphicsMessagesError;
-use crate::geometry::Rectangle;
+use crate::geometry::InclusiveRectangle as Rectangle;
```

Comment:
I feel we should use the actual name of the struct instead of `Rectangle`, to avoid confusion with the trait (and confusion between inclusive and exclusive rectangles also). What do you think?

Reply:
Yea, I planned to rename this, this is temporary to reduce the current PR diff

Reply:
I understand, makes sense!